contracts/RiskSteward/RiskStewardReceiver.sol

```diff
@@ -0,0 +1,322 @@
+     */
+    error UpdateIsExpired();
+
+    /**
+     * @notice Thrown when an update has already been processed
+     */
+    error ConfigAlreadyProcessed();
+
+    /**
+     * @notice Thrown when the debounce period hasn't passed for applying an update to a specific market/ update type
+     */
+    error UpdateTooFrequent();
+
+    /**
+     * @notice Thrown when the new value of an update is our of range
+     */
+    error UpdateNotInRange();
+
+    /**
+     * @notice Thrown when a config that is not implemented is set
+     */
+    error UnsupportedUpdateType();
+
+    /**
+     * @notice Flag to pause/unpause update processing
+     */
+    bool public paused;
+
+    /**
+     * @notice Event emitted when pause state is changed
+     */
+    event PauseStateChanged(bool newState);
+
+    /**
+     * @notice Error thrown when contract is paused
+     */
+    error ContractPaused();
+
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address riskOracle_, address corePoolComptroller_) {
+        require(riskOracle_ != address(0), "Risk Oracle address must not be zero");
+        require(corePoolComptroller_ != address(0), "Core Pool Comptroller address must not be zero");
+        RISK_ORACLE = IRiskOracle(riskOracle_);
+        CORE_POOL_COMPTROLLER = ICorePoolComptroller(corePoolComptroller_);
+        _disableInitializers();
+    }
+
+    function initialize() external initializer {
+        __Ownable2Step_init();
+    }
+
+    /**
+     * @notice Sets the risk parameter config for a given update type
+     * @param updateType The type of update to set the config for
+     * @param debounce The debounce period for the update
+     * @param maxIncreaseBps The max increase bps for the update
+     * @param isRelative Whether the max increase bps is relative or absolute
+     */
+    function setRiskParameterConfig(
+        string calldata updateType,
+        uint256 debounce,
+        uint256 maxIncreaseBps,
+        bool isRelative
+    ) external onlyOwner {
+        riskParameterConfigs[updateType] = RiskParamConfig({
+            active: true,
+            debounce: debounce,
+            maxIncreaseBps: maxIncreaseBps,
+            isRelative: isRelative
+        });
+    }
+
+    /**
+     * @notice Gets the risk parameter config for a given update type
+     * @param updateType The type of update to get the config for
+     * @return The risk parameter config for the given update type
+     */
+    function getRiskParameterConfig(string calldata updateType) external view returns (RiskParamConfig memory) {
+        return riskParameterConfigs[updateType];
+    }
+
+    /**
+     * @notice Toggles the active status of a risk parameter config
+     * @param updateType The type of update to toggle the config for
+     */
+    function toggleConfigActive(string calldata updateType) external onlyOwner {
+        riskParameterConfigs[updateType].active = !riskParameterConfigs[updateType].active;
+    }
```

Comment:
I would check if there is an entry for `riskParameterConfigs[updateType]` before updating `active`. Otherwise, we could have inconsistencies in that mapping. Alternatively,

* we could remove this function and force users to use `setRiskParameterConfig`, providing every parameter every time
* we could invoke `setRiskParameterConfig` from this function, simply flipping the `active` flag